tests/testthat/test-printing.R

```diff
@@ -340,6 +340,7 @@ test_that("Various Printing things work", {
 
   table_structure(tab, detail = "subtable") ## treestruct(tab)
   table_structure(tab, detail = "row") ## treestruct(tab)
+  coltree_structure(tab)
```

Comment:
We need some more details about these functions

Reply:
coltree_structure is intended for us (developers/maintainers) not end users,  I'm not sure exhaustive detail would be much help honestly. The way it displays the coltree structure is arbitrary but useful